client/src/components/styles/Amenities.styles.js

```diff
@@ -29,14 +40,16 @@ export const CategoryContainer = styled.div`
 
 export const CategoryHead = styled.div`
```

Comment:
The cathead and catitemcontainer have the same things mainly, I think you can reuse this by just using a props conditional for the differences.

eg: 
<category item>
border-bottom: ${props => props.item? XXX : XXX}